### ISSUE TITLE
Add checkboxes also to Sharli, Mastodon and Friendica

### ISSUE THREAD

**Issue by Sav22999** [Mte90/Share-Backported#146]
When they are enabled, so the textbox relative to that service is enabled as well, otherwise it should be disabled.

**Comment by Mte90** [comment]
As today, it use the fact that if the textarea is not empty show the social, so we can avoid the checkbox.
Implementing this for me will create some conflicts with previous installation.

**Comment by Sav22999** [comment]
Yes, but it's incoherent because other services are the checkbox to enable/disable.

**Comment by Mte90** [comment]
I agree that is incoherent but I am not sure if it is the case to do that change...
